barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/circuit_builder_base.hpp

```diff
@@ -12,25 +11,14 @@
 namespace proof_system {
 static constexpr uint32_t DUMMY_TAG = 0;
 
-template <typename Arithmetization> class CircuitBuilderBase {
+template <typename FF_> class CircuitBuilderBase {
   public:
-    using FF = typename Arithmetization::FF;
+    using FF = FF_;
```

Comment:
Any reason to not just call it FF instead of doing this using?

Reply:
This is to allow access to FF via e.g. `Standard::FF` from outside of this class. This has always felt a bit hacky but I'm not aware of another way to access the template param.

Reply:
Gotcha, might not be anything better then, thanks.